Detections/MultipleDataSources/NOBELIUM_IOCsMay2021.yaml

```diff
@@ -81,10 +109,13 @@ query:  |
   | extend IPMatch = case(SourceIP in (IPList), "SourceIP", DestinationIP in (IPList), "DestinationIP", MessageIP in (IPList), "Message", RequestURL in (domains), "RequestUrl", SourceIP in (ips), "SourceIP", DestinationIP in (ips), "DestinationIP", MessageIP in (IPList), "Message", "NoMatch") 
   | extend timestamp = TimeGenerated, IPCustomEntity = case(IPMatch == "SourceIP", SourceIP, IPMatch == "DestinationIP", DestinationIP, IPMatch == "Message", MessageIP, "NoMatch"), AccountCustomEntity = SourceUserID
   ),
-  (DnsEvents
-  | where IPAddresses in (IPList) or IPAddresses in (ips) or Name in~ (domains)  
-  | extend DestinationIPAddress = IPAddresses, DNSName = Name, Host = Computer
-  | extend timestamp = TimeGenerated, IPCustomEntity = DestinationIPAddress, HostCustomEntity = Host
+  (_Im_Dns (domain_has_any=todynamic(domain_list))
+  | extend DestinationIPAddress = DnsResponseName, DNSName = DnsQuery, Host = Dvc
+  | extend timestamp = TimeGenerated, IPCustomEntity = SrcIpAddr, HostCustomEntity = Host
+  ),
+  (_Im_Dns (response_has_any_prefix=todynamic(full_ip_list))
+  | extend DestinationIPAddress = DnsResponseName, DNSName = DnsQuery, Host = Dvc
+  | extend timestamp = TimeGenerated, IPCustomEntity = SrcIpAddr, HostCustomEntity = Host
   ),
```

Comment:
Remove Event, DeviceNetworkEvent. Add WindowsFirewall to our list for the vendor

Reply:
removed Event and Device.